### ISSUE TITLE
Easily add Aspire orchestrator support in existing projects

### ISSUE THREAD

**Issue by thimok** [rafaelldi/aspire-plugin#182]
In Visual Studio, if you right click a project, under 'add' there is '.NET Aspire Orchestrator Support'.

This adds the project to the apphost as a project reference, adds the AddServiceDefaults() extension method in the Program file of the project, etc.

It would be great if this feature could be added to the plugin.

**Comment by Foxtrek64** [comment]
This is also a requirement for following the Aspire course on Microsoft Learn when using Rider.

**Comment by RSReswin** [comment]
how to add .NET Aspire Orchestrator Support in JetBrains Rider ?

**Comment by rafaelldi** [comment]
@RSReswin You could manually add a new Aspire App Host project to the solution.

<img width="541" alt="Image" src="https://github.com/user-attachments/assets/233fbc11-3de4-4ae8-9772-e642118bcea1" />

<img width="677" alt="Image" src="https://github.com/user-attachments/assets/7109f04c-635c-48a5-8c3e-13429246ea2e" />

**Comment by RSReswin** [comment]
@rafaelldi i am not asking about how to add aspire project in solution, i am ask about how to add Orchestrator Support ?

**Comment by rafaelldi** [comment]
@RSReswin What do you mean by "Orchestrator Support"?

**Comment by RSReswin** [comment]
@rafaelldi https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/app-host-overview?tabs=docker

![Image](https://github.com/user-attachments/assets/7097087b-c0e1-459e-a7e8-0a732a42f61a)

**Comment by rafaelldi** [comment]
@RSReswin This action creates an Aspire App Host project in the solution. If you're asking if it's possible to create everything in one click, no, unfortunately, that's not supported right now.

**Comment by thimok** [comment]
> [@RSReswin](https://github.com/RSReswin) This action creates an Aspire App Host project in the solution. If you're asking if it's possible to create everything in one click, no, unfortunately, that's not supported right now.

Not exactly. If you already have an orchestrator in your solution, this context action:
* Adds your selected project as a reference to your orchestrator
* Adds your selected project to the Program.cs to the DistributedApplicationBuilder
* Adds the ServiceDefaults project as a reference to your selected project
* Adds the .AddServiceDefaults() extension method in the Program.cs of your selected project

It does quite a few things to _enlist_ your project in the orchestrator. Of course, if you don't have an Aspire orchestrator yet, it will also create that project (and the ServiceDefaults project).

**Comment by rafaelldi** [comment]
@thimok Yes, thanks. I understand that this action may be pretty convenient, but, unfortunately, I don't have enough resources for now to support it. I meant that it can be done manually now (not as fast or convenient, of course). Supporting this action is in my plans but after other more urgent tasks.
